models/pancakeswap/pancakeswap_trades.sql

```diff
@@ -10,6 +10,8 @@
 {% set pancake_models = [
     ref('pancakeswap_v2_bnb_trades')
 ,   ref('pancakeswap_v2_ethereum_trades')
+,   ref('pancakeswap_v2_bnb_mmpool')
+,   ref('pancakeswap_v2_ethereum_mmpool')
```

Comment:
Hi @chef-seaweed,
do you think we need to rename `pancakeswap_v2_bnb_trades` and `pancakeswap_v2_ethereum_trades`
to `pancakeswap_v2_bnb_amm_trades` and `pancakeswap_v2_ethereum_amm_trades`?

Also do you think we need to rename `pancakeswap_v2_bnb_mmpool` and `pancakeswap_v2_ethereum_mmpool` to `pancakeswap_v2_bnb_mmpool_trades` and `pancakeswap_v2_ethereum_mmpool_trades`?

Reply:
Yes please.